src/main/java/com/google/graphgeckos/dashboard/storage/DatastoreRevisionEntity.java

```diff
@@ -0,0 +1,109 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.graphgeckos.dashboard.storage;
+
+import com.google.cloud.Timestamp;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * An intermediary container used by the Spring - Google Cloud Datastore integration,
+ * allowing easy creation/update of new entities in the database. It is created using
+ * a commit's metadata from git, and updated using
+ * {@link com.google.graphgeckos.dashboard.storage#Builder Builder} objects, which add
+ * compilation information from each buildbot. The scope of an instance is supposed to
```

Comment:
"The scope of an instance is supposed to be contained inside a single method" - can I get a clarification on what the instance is in this context?

Reply:
For example, inside the createRevisionEntry method, this class is only used as follows: `DatastoreRevisionEntity newEntity = new DatastoreRevisionEntity(entryData);` followed immediately by `storage.save(newEntity);`, after which the instance's lifetime is done. Roughly the same happens for the updateRevisionEntry usecase.

Reply:
And to directly address your question, an instance here is a constructed DatastoreRevisionEntity object.